src/main/scala/cognite/spark/v1/CDFSparkException.scala

```diff
@@ -0,0 +1,27 @@
+package cognite.spark.v1
+
+trait CDFSparkException {}
+
+class CDFSparkInternalException(message: String) extends Exception(message) with CDFSparkException {
```

Comment:
I think we can drop `Internal` from this name.

Reply:
That would conflict with the trait. We could rename the trait to something else, but I'm not sure what.

Reply:
Could we make CdfSparkException be a class and have CdfSparkIllegalArgumentException inherit from it?

Reply:
We can do that, however, then `CdfSparkIllegalArgumentException` can not inherit from `IllegalArgumentException` due to multiple inheritance. But, we probably don't need to inherit from `IllegalArgumentException`.

Reply:
Because many of those messages like "Granularity specified but no aggregation requested" are not actually internal at all, so I'm not a big fan of that name.

Reply:
I have removed the trait now. Had to change some tests to expect `CdfSparkIllegalArgumentException` instead of `IllegalArgumentException`.